app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -1,95 +1,119 @@
-module ManageIQ::Providers
-  class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
-    require_nested :MiddlewareDeployment
-    require_nested :MiddlewareServer
-    require_nested :RefreshParser
-    require_nested :RefreshWorker
-    require_nested :Refresher
-
-    include AuthenticationMixin
-
-    DEFAULT_PORT = 80
-    default_value_for :port, DEFAULT_PORT
-    has_many :middleware_servers, :foreign_key => :ems_id, :dependent => :destroy
-    has_many :middleware_deployments, :foreign_key => :ems_id, :dependent => :destroy
-
-    def verify_credentials(_auth_type = nil, options = {})
-      begin
-
-        # As the connect will only give a handle
-        # we verify the credentials via an actual operation
-        connect(options).list_feeds
-      rescue => err
-        raise MiqException::MiqInvalidCredentialsError, err.message
-      end
-
-      true
+class ManageIQ::Providers::Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
```

Comment:
I'm a fan of this change.  But, can you save it for a different PR?  It makes it harder to tell what actually changed in this file in this PR.